trollsched/schedule.py

```diff
@@ -42,12 +42,8 @@
 
 logger = logging.getLogger(__name__)
 
-# shortest allowed pass in minutes
-MIN_PASS = 4
-
 # name/id for centre/org creating schedules
-# CENTER_ID = "SMHI"
-CENTER_ID = "DWD-OF"
+CENTER_ID = "SMHI"
```

Comment:
I thought we used schedule.CENTER_ID ?

Reply:
I hadn't looked at your commit to develop, this line is obsolete then.